pypeit/par/pypeitpar.py

```diff
@@ -543,10 +543,10 @@ def __init__(self, method=None, maxshift=None, spectrum=None):
 
         # Fill out parameter specifications.  Only the values that are
         # *not* None (i.e., the ones that are defined) need to be set
-        defaults['method'] = 'boxcar'
+        defaults['method'] = 'skip'
```

Comment:
Skip means no correction. I think the problem is that the parsets don't work well if you instantiate to None here, because I think you cannot read in None from a cfg file. Perhaps @kbwestfall can comment. Anyway, I changed to 'skip' for that reason. Happy to change to something else like 'None', but I think we may have an issue with the None value being initiated in code.